.deploy/Dockerfile

```
@@ -7,7 +7,33 @@
-    ARG DOPPLER_PROJECT
-    ARG DOPPLER_CONFIG
+ARG DOPPLER_TOKEN
+ARG DOPPLER_PROJECT="yo-mama"
+ARG DOPPLER_CONFIG="prod"
+ARG DJANGO_CONFIGURATION="Production"
+
+ENV DJANGO_CONFIGURATION=${DJANGO_CONFIGURATION}
+
+WORKDIR /src/app
+
+    chmod 770 /src/app/.doppler
```

Hadolint (2.12.0)

[error] unexpected 'c'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)